File: EstructurasControl/Repetitivas/DoWhile/Csharp/ejercicio2.cs
using System;

class RepetitivaDoWhile2
{
    static void Main()
    {
        int opcion;

        // Bucle do-while
        do
        {
            Console.WriteLine("\n=== Menú Principal ===");
            Console.WriteLine("1. Saludar");
            Console.WriteLine("2. Despedirse");
            Console.WriteLine("3. Salir");
            Console.Write("Elige una opción: ");
            opcion = Convert.ToInt32(Console.ReadLine());

            if (opcion == 1)
            {
                Console.WriteLine("¡Hola! 👋");
            }
            else if (opcion == 2)
            {
                Console.WriteLine("¡Adiós! 👋");
            }

        } while (opcion != 3);

        Console.WriteLine("Programa finalizado.");
    }
}

File: EstructurasControl/Repetitivas/DoWhile/Python/ejercicio2.py
print("=== Menú Principal ===")

while True:
    print("\n1. Saludar")
    print("2. Despedirse")
    print("3. Salir")
    
    opcion = int(input("Elige una opción: "))

    if opcion == 1:
        print("¡Hola! 👋")
    elif opcion == 2:
        print("¡Adiós! 👋")
    elif opcion == 3:
        print("Programa finalizado.")
        break  # condición de salida
    else:
        print("Opción no válida, intenta de nuevo.")
